### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by jbronn**
Here's the documentation PR.  Please read over, don't be afraid to be a spelling/grammar pedant -- I want to be as clear as possible and know there's room for improvement.

I need to flesh out  the shell and installation documentation (by the time you review this, the install from S3 section, #161, should at least be finished).

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Release Process
+
+## Requirements
+
+* GPG Key: You'll the Hootenanny packaging release public and private
```
Comment (sputnam) [inline docs/release.md]: Double check wording on this line.

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+   This should generate the following files in `RPMS`:
+
+   * `noarch/hootenanny-autostart-$HOOT_VERSION_GEN-1.el7.noarch.rpm`
+   * `noarch/hootenanny-core-deps-$HOOT_VERSION_GEN-1.el7.noarch.rpm`
+   * `noarch/hootenanny-core-devel-deps-$HOOT_VERSION_GEN-1.el7.noarch.rpm`
+   * `noarch/hootenanny-services-devel-deps-$HOOT_VERSION_GEN-1.el7.noarch.rpm`
+   * `x86_64/hootenanny-core-$HOOT_VERSION_GEN-1.el7.x86_64.rpm`
+   * `x86_64/hootenanny-services-ui-$HOOT_VERSION_GEN-1.el7.x86_64.rpm`
+
+   Note: The default the release value in the RPM version is `1`.
```
Comment (sputnam) [inline docs/release.md]: Extra `the`?

File: docs/release.md (hunk 1 above)
Comment (jbronn) [reply docs/release.md]: Fixed in 4052721.

File: docs/release.md (hunk 2 above)
Comment (jbronn) [reply docs/release.md]: Fixed in 4052721.

File: docs/setup.md [hunk 3]
```diff
@@ -1,53 +1,62 @@
 # Setup
 
-In order to generate Hootenanny RPMs, you'll need RPM tools, Docker, Vagrant, and `make`.
-Fortunately, scripts are included to automate the setup process for Linux users.
-The scripts assume:
+In order to generate Hootenanny RPMs, you'll need RPM tools, Docker, Vagrant,
```
Comment (Cellington1) [inline docs/setup.md]: Nit-picky here and I may be wrong, but the Hootenanny RPMs can be generated with RPM tools, Docker and make alone, correct?

Comment (jbronn) [reply docs/setup.md]: Good point.  I'll clarify that it's preferred to use Vagrant and `make`, and point to the shell documentation (which needs to exist first).

Comment (jbronn) [reply docs/setup.md]: Fixed in 34d9ddf.

**Pull request merged**